RebornCore/src/main/java/reborncore/common/blockentity/MachineBaseBlockEntity.java

```diff
@@ -117,7 +126,8 @@ public boolean isMultiblockValid() {
 	public void writeMultiblock(MultiblockWriter writer) {}
 
 	public void syncWithAll() {
-		if (world == null || world.isClient) { return; }
+		if (world == null || world.isClient|| tickTime < lastTickedSync + 20) { return; }
```

Comment:
What is casuing this to sync so much?

If its beign called everytick or something that should be sorted at the root of the issue imo.

Reply:
Its part of packet issue, mostly storage units, are being updated once altogether and causing packet bottleneck for certain players, generating packet 'spikes'... so this is not really the solution.

 I found that block entities are having same interval and same starting point, which was not spreading out packets over time(also mspt too).